### PR TITLE
[Master - Bug 11163] Uninitialized value warning in Excel output in AgentTicketSearch

### PR DESCRIPTION
**Pull request by s7design**
http://bugs.otrs.org/show_bug.cgi?id=11163

### CODE REVIEW

**Comment by mgruner** [comment]
Hm. When I do a simple ticket search and output to Excel, your patch destroys the excel file. It is cut after the first emtpy cell, no further cells are being populated. Does it work for you?

File: Kernel/System/CSV.pm [hunk 1]
```diff
@@ -119,7 +119,9 @@ sub Array2CSV {
         my @ColumnLengths;
         my $Row = 0;
         for my $DataRaw ( \@Head, @Data ) {
+            DATARAW:
```
Comment (mgruner) [inline Kernel/System/CSV.pm]: This has the wrong name, it should be COL: if it is placed correctly here.

**Comment by s7design** [comment]
Hi @mgruner ,

I hope that it is works now.

Regards
Zoran

**Comment by mgruner** [comment]
Merged, thanks!

**Pull request closed without merging**